css/style.css

```diff
@@ -0,0 +1,120 @@
+.calc {
+    position: relative;
+    width: 411px;
+    height: 731px;
+    margin: auto;
+    background: #FFFFFF;
```

Comment:
lets use css variables for all colors https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties